src/literal.h

```diff
@@ -33,25 +33,27 @@ class Literals;
 struct ExceptionPackage;
 
 class Literal {
+
+private:
```

Comment:
I don't think we need the `private` here, since the default visibility for `class`es is already `private`.